packages/fx-core/src/plugins/solution/fx-solution/v2/deploy.ts

```diff
@@ -223,7 +231,18 @@ export async function deploy(
             }
           });
       } else {
-        ctx.userInteraction.showMessage("info", msg, false);
+        if (isDeployAADManifestFromVSCode) {
+          ctx.userInteraction
+            .showMessage("info", msg, false, getLocalizedString("core.deploy.aadManifestLearnMore"))
```

Comment:
What will happen in cli? Will there be a button to click in cli?

Reply:
Will have follow up after offline discussion.

Reply:
This logic is only for VSCode, for CLI part, I will create a task and discuss with Junjie how to update it in the future PR.